dask_planner/Cargo.toml

```diff
@@ -22,6 +22,7 @@ mimalloc = { version = "*", default-features = false }
 parking_lot = "0.12"
 async-trait = "0.1.41"
 sqlparser = "0.18"
+serde_json = { version="1.0.79", features = ["std"] }
```

Comment:
Needed to add this to my environment which already had rust/cargo etc installed via rustup. Things work fine on a machine where we get rust only from conda. Not sure why that is